drivers/qemu.c

```diff
@@ -0,0 +1,20 @@
+#include "../kinc/qemu.h"
+#include "../kinc/serial.h"
+#include "../kinc/api.h"
+
+uint32 qemu_init_debug(){
+    init_serial(QEMU_SERIAL_PORT);
+    return 1;
```

Comment:
mmm i would say here, don't return anythign at all since it alway return 1.

Even because in the init_serial the return 1 means that there is an error, and in this function means kind of: "all ok", so i would souggest just make qemu_init_debug return void